### PR TITLE
org index migration

### PR DESCRIPTION
**Pull request by pdeziel**
### Scope of changes

This adds a migration script to add the missing org index entries in the tenant database.

Fixes SC-16501

### Type of change

- [ ] new feature
- [ ] bug fix
- [ ] documentation
- [ ] testing
- [ ] technical debt
- [x] migration

### Acceptance criteria

Describe how reviewers can test this change to be sure that it works correctly. Add a checklist if possible.

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [ ]  Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [ ]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

File: cmd/tenant/main.go [hunk 1]
```diff
@@ -29,6 +37,33 @@ func main() {
 			Action:   serve,
 			Flags:    []cli.Flag{},
 		},
+		{
+			Name:     "db:list",
+			Usage:    "list all keys in a tenant namespace",
```
Comment (pdeziel) [inline cmd/tenant/main.go:42]: We might want to include this in main - I found it pretty useful for debugging, especially since the IDs are ordered.

Comment (bbengfort) [reply cmd/tenant/main.go:42]: sounds good!

File: cmd/tenant/main.go [hunk 2]
```diff
@@ -29,6 +37,33 @@ func main() {
...
+				&cli.StringFlag{
+					Name:     "namespace",
+					Aliases:  []string{"n"},
+					Usage:    "namespace to list keys for",
+					Required: true,
+				},
+			},
+		},
+		{
+			Name:     "db:migrate:orgs",
```
Comment (bbengfort) [inline cmd/tenant/main.go:55]:
```suggestion
			Name:     "db:reindex",
```

Re-indexing may be something that we need to do periodically across all of our indexes; so I suggest we merge this as a general purpose utility and identify other indexes that could be updated (completely) to fix problems.

File: cmd/tenant/main.go [hunk 3]
```diff
@@ -57,3 +92,221 @@ func serve(c *cli.Context) (err error) {
...
+
+func listKeys(c *cli.Context) (err error) {
+	var conf config.Config
+	if conf, err = config.New(); err != nil {
+		return cli.Exit(err, 1)
+	}
+
+	if err = db.Connect(conf.Database); err != nil {
+		return cli.Exit(err, 1)
+	}
```
Comment (bbengfort) [inline cmd/tenant/main.go:261]: I know this is a quick script; but if we're going to have these commands be general purpose utilities, we should have `Before` and `After` -- this doesn't close the connection to the DB either. I'll make those changes and push them up so we can move on to the migration.

File: cmd/tenant/main.go (hunk 2 above)
Comment (pdeziel) [reply cmd/tenant/main.go]: Good idea!

**Pull request merged**